packages/core/src/components/tabs2/_tabs2.scss

```diff
@@ -0,0 +1,64 @@
+// Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+/*
+JavaScript API
+
+<div class="pt-callout pt-intent-danger pt-icon-error">
+  <h5>Original `Tabs` API is deprecated since v1.10.0</h5>
+  The original `Tabs` API has been deprecated in v1.10.0 favor of the simpler and more flexible
+  `Tabs2` API described below. Documentation for the deprecated components can be found
+  [further below](#components.tabs.deprecated). The deprecated components will be removed in v2.0.
+</div>
+
+<div class="pt-callout pt-intent-primary pt-icon-info-sign">
+  <h5>Advantages of new API</h5>
```

Comment:
next time leave the comment on the suspect code. i fixed this locally but this comment may not disappear as i haven't touched _this_ `h5` line.